primitives/traits/src/lib.rs

```diff
@@ -562,4 +562,8 @@ pub trait ReadProofSubmittersInterface {
     fn get_proof_submitters_for_tick(
         tick_number: &Self::TickNumber,
     ) -> Option<BoundedBTreeSet<Self::ProviderId, Self::MaxProofSubmitters>>;
+
+    fn get_accrued_failed_proof_submissions(provider_id: &Self::ProviderId) -> Option<u32>;
+
+    fn clear_accrued_failed_proof_submissions(provider_id: &Self::ProviderId);
```

Comment:
This feels a bit out of place in a `ReadProofSubmittersInterface` (for the read part)

Reply:
Want to rename this to `ProofSubmittersInterface`